k9mail/src/main/java/com/fsck/k9/fragment/MessageListFragment.java

```diff
@@ -1536,16 +1512,16 @@ private void handleSwipe(final MotionEvent downMotion, final boolean selected) {
     }
 
     private int listViewToAdapterPosition(int position) {
-        if (position > 0 && position <= mAdapter.getCount()) {
-            return position - 1;
+        if (position >= 0 && position < mAdapter.getCount()) {
+            return position;
         }
 
         return AdapterView.INVALID_POSITION;
     }
 
     private int adapterToListViewPosition(int position) {
```

Comment:
I thought the same, but I was not sure. I didn't know if `AdapterView.INVALID_POSITION` is something important...